dart/browser_event_demo.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:html';
```

Comment:
Most of `dart:html` is autogenerated from V8, so it isn't terribly interesting compared to JavaScript.